process/accounts/accounts_processor_test.go

```diff
@@ -117,18 +117,17 @@ func TestAccountsProcessor_ProcessAccounts_FourAddresses_TwoIdentical_ExpectTwoA
 func TestAccountsProcessor_ProcessAccounts_OneAddress_OneMetaChainShardAddress_ExpectOneAccount(t *testing.T) {
 	ap, _ := accounts.NewAccountsProcessor(&mock.ShardCoordinatorMock{})
 
+	addresses := generateAddresses(1)
 	tx := &schema.Transaction{
-		Receiver: []byte("adr1"),
+		Receiver: []byte("adr0"),
```

Comment:
Can we use `addresses[0]` instead of `adr0` so that in case the logic inside `generateAddresses`/`checkProcessedAccounts` changes, this test won't fail ?

Reply:
I cannot use the generateAddresses/checkProcessedAccounts because they will set the balance as `"0"`, and I need `""`